Standardize documentation sites

jsiegle opened the pull request:
- Update to latest version of `pydata-sphinx-theme` (`0.13.3`)
- Standardize sidebar font colors
- Standardize fonts (`h1` through `h6`) and link colors
- Put GitHub + Twitter + Discord links in navbar
- Standardize landing page
	- Remove large title
	- Change page title to "Home"
	- Start with large image
	- Page link cards in one row instead of 2 and remove borders
	- Remove license info 
- Add LICENSE file to source code (CC-BY-SA 4.0)
- Place search bar on landing page only; otherwise it's in the navbar
- Standardize logo font and size (and add template + instructions to `doc-template` repo)
- Standardize HTML page names (Page Title – Project Name)
- Make sure "Edit on GitHub" and "Show Source" links are visible

ChucklesOnGitHub commented:
This is really great, Josh. I had some comments of things I had noticed in the documentation a while back but I think this sorts them out. Some other suggestions here:
- Would it be possible to include fist line indentation to improve readability? e.g. 
https://stackoverflow.com/questions/68730912/first-line-indentation-in-sphinx
- Different height of navigation menus: ac board and twister are thick vs GUI and ONIX which are thinner
- Some GitHub logos on the docs go to the doc site while other go to the main open ephys github landing page. Make sure we want to link to the source code of the device?
- The downloads page on the ONIX documentation
- Put store link in navbar like the acq boards docs have https://open-ephys.org/
- Is it now possible to embed Bonsai workflows in all the docs sites? Does this go in the template somehow? It was done for ONIX and Commutator docs, and it would be useful for Acq Boards and upcoming Miniscope docs.

Things I had seen that are now taken care of as far as I can tell:
- Date of copyright has to be automatic (e.g. acq board did not say 2023)
- CC License for docs

Comment:
-Start with large image in landing page might not work for all docs (see Twister and Commutators for example). I noticed there were two ways in which things had been done:
- Landing page plus multiple sections (folders) each with their own landing page and subpages
- Or have them as in the twister docs, just linked pages to one main landing page. Twister3 docs does not have multiple pages under each tab in the navigation menu. It has the Contents at the top of each page and the “On this page” section on the right.
I guess the idea would be to unify this, is that the point?

jsiegle commented:
Thanks for the feedback, this is really helpful!

> Would it be possible to include first line indentation to improve readability? 

I think I prefer to add more space between each paragraph, rather than indenting the first line. I changed the `theme_overrides` files so the spacing is consistent across sites. Can you let me know if this looks better?

> Different height of navigation menus: ac board and twister are thick vs GUI and ONIX which are thinner

I'm not seeing this in my local builds. Can you post a screenshot?

> Some GitHub logos on the docs go to the doc site while other go to the main open ephys github landing page. Make sure we want to link to the source code of the device?

Good catch, this is fixed now.

> Put store link in navbar like the acq boards docs have https://open-ephys.org/

I added a "Purchase" link to the store for any devices, and a "Download" link for the GUI.

> Is it now possible to embed Bonsai workflows in all the docs sites? Does this go in the template somehow? It was done for ONIX and Commutator docs, and it would be useful for Acq Boards and upcoming Miniscope docs.

I haven't changed anything related to this – can you look into what's needed?

> Start with large image in landing page might not work for all docs (see Twister and Commutators for example). 

Ideally we can get new images with the correct aspect ratio for these docs (ShuttleDrive too).

> I noticed there were two ways in which things had been done:
> - Landing page plus multiple sections (folders) each with their own landing page and subpages
> - Or have them as in the twister docs, just linked pages to one main landing page. Twister3 docs does not have multiple pages under each tab in the navigation menu. It has the Contents at the top of each page and the “On this page” section on the right.
> I guess the idea would be to unify this, is that the point?

I think the Commutator docs are the only exception to this – instead of having a "User manual" and "Assembly/developer guide", it has main sections for different types of commutators (with the SPI commutator currently hidden). Depending on how similar the Coax and SPI commutators end up being, it may be easier to switch to the standard method of organization and include notes on each page wherever the two designs differ.
